pkg/api/aim/runs.go

```diff
@@ -171,16 +171,15 @@ func GetRunMetrics(c *fiber.Ctx) error {
 		return fiber.NewError(fiber.StatusUnprocessableEntity, err.Error())
 	}
 
-	metricKeysMap, contexts := make(fiber.Map, len(b)), make([]types.JSONB, 0, len(b))
+	metricKeysMap := make(map[string]types.JSONB, len(b))
 	for _, m := range b {
 		if m.Context != nil {
 			serializedContext, err := json.Marshal(m.Context)
 			if err != nil {
 				return fiber.NewError(fiber.StatusUnprocessableEntity, err.Error())
 			}
-			contexts = append(contexts, serializedContext)
+			metricKeysMap[m.Name] = serializedContext
```

Comment:
it looks to me like requesting 2 metrics with the same key and a different context won't work anymore with this code.

e.g. the following request would result in a `metricKeysMap` with only one entry:
```json
[
    {
        "context": {
            "mode": "validation"
        },
        "last_value": 0.1,
        "name": "metric1"
    },
    {
        "context": {
            "mode": "train"
        },
        "last_value": 0.1,
        "name": "metric1"
    }
]
```

Reply:
could use `models.Metric.UniqueKey()` if they always belong to the same run